{
  "timestamp": "2025-08-06T08:26:07.428Z",
  "network": "bonsoleil",
  "deployer": "0xCf20a6EcBBedB403DB466D669229d9Ee379C433f",
  "contracts": {
    "itemBank1": "0x125F456DD6633E9E698FC084Ac669E08B133daaA",
    "itemBank2": "0x86F003F07704F2939FB5eD40e16f13521D6c930E",
    "itemBankV3": "0x2A76B76D2024bDf8018459cbf789A5B0bd277Fdc",
    "monsterBank1": "0xe4fc503255De4F536BBC43Fe339843F88E2e1E66",
    "monsterBank2": "0xE7b23Cc85D6D9b24C4203F26A97DF7A80F9FA047",
    "monsterBankV3": "0x41717f85854D631F7cb856D634fBf396beb68C93",
    "backgroundBank": "0x1103d4c6108705610B08c2FFA696e3589579931E",
    "effectBank": "0xA1D8e09E0F6559e992e2882555F60bE67F4e73a9",
    "composerV5": "0x4036C601b705d68cA91281fdD1D665236ABbFde7",
    "metadataV5": "0xbFD4e357e0552Dd26b1A4877a5DB8DDC2E4198C4",
    "bankedNFT": "0xb0C8bCef9bBEd995b18E0fe6cB7029cB7c90E796"
  },
  "error": "missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data=\"0x\", transaction={\"from\":\"0xCf20a6EcBBedB403DB466D669229d9Ee379C433f\",\"gasLimit\":{\"type\":\"BigNumber\",\"hex\":\"0x4c9ed8\"},\"to\":\"0xbFD4e357e0552Dd26b1A4877a5DB8DDC2E4198C4\",\"data\":\"0xa574cea40000000000000000000000000000000000000000000000000000000000000000\",\"accessList\":null}, error={\"name\":\"ProviderError\",\"_stack\":\"ProviderError: execution reverted\\n    at HttpProvider.request (/Users/goodsun/Desktop/generativeNft/monster/tragedyContract/formal_procedure_p1/node_modules/hardhat/src/internal/core/providers/http.ts:116:21)\\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\\n    at EthersProviderWrapper.send (/Users/goodsun/Desktop/generativeNft/monster/tragedyContract/formal_procedure_p1/node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)\",\"code\":-32000,\"_isProviderError\":true}, code=CALL_EXCEPTION, version=providers/5.8.0)"
}